include/zephyr/drivers/video.h

```diff
@@ -96,6 +96,16 @@ struct video_caps {
 	uint8_t min_vbuf_count;
 };
 
+/**
+ * @brief video_frame_fragmented_status enum
+ *
+ * Indicates the receiving status of fragmented frames.
+ */
+enum video_frame_fragmented_status {
+	VIDEO_BUF_FRAG,
+	VIDEO_BUF_EOF,
+};
```

Comment:
```suggestion
#define VIDEO_BUF_FRAG BIT(0)
#define VIDEO_BUF_EOF  BIT(1)
```

I think this is what @loicpoulain suggested [here](https://github.com/zephyrproject-rtos/zephyr/pull/66994#issuecomment-1905458088), but I could be wrong!